### PR TITLE
Travis/badge list accordion

### PR DESCRIPTION
**Pull request by traversng**
https://www.loom.com/share/94d3aaea7e9d4d889b3c27e1808dbcbb?sid=4ae5ccfb-8456-46c7-9baf-46005f889866

### CODE REVIEW

File: src/app/badges/page.tsx [hunk 1]
```diff
@@ -26,46 +27,34 @@ export default function Badges() {
           {error && <div>Error...</div>}
           {badges && (
             <>
-              <Stack
+              <Box
```
Comment (siwan-cb) [inline src/app/badges/page.tsx:30]: why box?

File: src/components/Badges/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import AccordionPill from './AccordionPill';
+
+export default function CustomAccordion({
+  children,
+  title,
+}: {
+  title: string;
+  children?: React.ReactNode;
```
Comment (siwan-cb) [inline src/components/Badges/Accordion.tsx:17]: /nit just `ReactNode`

File: src/components/Badges/Accordion.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}) {
+  // State for the expanded accordion panel
+  const [expanded, setExpanded] = useState<string | false>(false);
```
Comment (siwan-cb) [inline src/components/Badges/Accordion.tsx:20]: why not keep this as boolean?
```suggestion
  const [isExpanded, setIsExpanded] = useState<false>(false);
```

Comment (siwan-cb) [reply src/components/Badges/Accordion.tsx:20]: I don't see a reason to support a string type for this state

File: src/components/Badges/Accordion.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+  // Handle changing the expanded accordion panel
+  const handleChange =
+    (panel: string) => (event: React.SyntheticEvent, isExpanded: boolean) => {
+      setExpanded(isExpanded ? panel : false);
+    };
+
+  return (
+    <div>
```
Comment (siwan-cb) [inline src/components/Badges/Accordion.tsx:29]: you don't need this

File: src/components/Badges/AccordionPill.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
```
Comment (siwan-cb) [inline src/components/Badges/AccordionPill.tsx:1]: no * import. Import only thing that you need.

File: src/components/Badges/AccordionPill.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+import { Box } from '@mui/material';
+
+function AccordionPill(props: any) {
```
Comment (siwan-cb) [inline src/components/Badges/AccordionPill.tsx:4]: can you provide a type instead of any?

Comment (siwan-cb) [reply src/components/Badges/AccordionPill.tsx:4]: or is this until we have something more defined later?

File: src/components/Badges/AccordionPill.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+        borderRadius: '6.25rem', // 100px
+        backgroundColor: 'var(--Black, #151515)',
+        padding: '4px 8px',
+        font: '400 14px/17px "Coinbase Mono", -apple-system, Roboto, Helvetica, sans-serif',
+        '@media (max-width: 991px)': {
+          whiteSpace: 'initial',
+        },
+      }}
+    >
+      4/16
```
Comment (siwan-cb) [inline src/components/Badges/AccordionPill.tsx:20]: I guess we will get this from API later.

File: src/app/badges/page.tsx (hunk 1 above)
Comment (siwan-cb) [reply src/app/badges/page.tsx:30]: no reason to change it to box when Stack is essentially a box with direction and we aren't using a direction here.

Comment (siwan-cb) [reply src/app/badges/page.tsx:30]: no strong reference here.

Comment (traversng) [reply src/app/badges/page.tsx:30]: stack has some default styles I rather not need to override

**Pull request merged**